### ISSUE TITLE
how to run it?

### ISSUE THREAD

**Issue by pkunico** [dotSlashLu/kibana-4.1.4-auth#1]
run  src\server\bin\kibana.bat   can't start   reports: can't find folder
need npm to build?

**Comment by dotSlashLu** [comment]
Yes, this repo is for development, you'll have to build it yourself. You'll need `npm` and several other dependencies installed by `npm` to build it.

I'll seek a way to release the built package this weekend, stay tuned if you don't want to build it yourself.

**Comment by dotSlashLu** [comment]
You can also build it yourself by `grunt`:
1. install `npm`
2. `npm install -g grunt-cli`
3. `grunt build`

Then the built version will be located in `build/dist`

**Comment by pkunico** [comment]
release the built package will be a good idea!!
it still can't build,error report as below:

D:\kibana-4.1.4-auth>grunt build
Running "get_build_props" task
Fatal error: spawn sh ENOENT
